yt_dlp/extractor/eplus.py

```diff
@@ -9,7 +9,10 @@
 class EplusIbIE(InfoExtractor):
     IE_NAME = 'eplus:inbound'
     IE_DESC = 'e+ (イープラス) overseas'
-    _VALID_URL = r'https?://live\.eplus\.jp/ex/player\?ib=(?P<id>(?:\w|%2B|%2F){86}%3D%3D)'
+    _VALID_URL = [
+        r'https?://live\.eplus\.jp/ex/player\?ib=(?P<id>(?:\w|%2B|%2F){86}%3D%3D)',
+        r'https?://live\.eplus\.jp/(?P<id>sample)',
```

Comment:
I only know very little about the site, so bear with me.
As I understand it, `https://live.eplus.jp/ex/player?ib=`, are pre-authenticated URLs. Looking around, I found lots of URLs like this: `https://live.eplus.jp/2612511` that redirect to a login-in page. Are these the same playback pages, but unauthenticated? If yes, could support be added through `--cookies-from-browser`?

Reply:
I've already heard about the "`https://live.eplus.jp/\d+`" links, but I currently have no way to analyze them. Maybe your request should be put into a new issue about this extractor, please?

> Looking around, I found lots of URLs like this:...

Is there somewhere that contains those URLs? I googled but found nothing.

Reply:
I initially found them using the [Wayback Machine](https://web.archive.org/web/*/https://live.eplus.jp/*). Tracking them down further, a fair number of them come from twitter.

Reply:
> I initially found them using the [Wayback Machine](https://web.archive.org/web/*/https://live.eplus.jp/*). Tracking them down further, a fair number of them come from twitter.

Thanks, I got it.

Could you please file a "Site support request"? Content recoreded here is not easy to be found in the future.

Reply:
What do you think of optimistically adding:
```suggestion
        r'https?://live\.eplus\.jp/(?P<id>sample|\d+)',
```
And then this for the webpage download?
```python
webpage_handle = self._request_webpage(url, video_id, headers={'User-Agent': self._USER_AGENT})
if '/member/auth' in webpage_handle.url:
    self.raise_login_required()
webpage = webpage_handle.read().decode()
```
Assuming these pages are the same as the pre-authenticated ones, this should work. Though I'm not super convinced, this is better than not supporting the sites at all. Curious what you think.